test/integration/test_disco_accounts.py

```diff
@@ -1,72 +1,90 @@
 """
 Integration tests for disco_accounts.py
 """
-import os
+from random import randint
 from test.helpers.integration_helpers import IntegrationTest
 
 
 TEST_ACCOUNT_NAME = "test_account"
-TEST_ACCOUNT_S3_USER_KEY = "accounts/users/%s" % TEST_ACCOUNT_NAME
-TEST_ACCOUNT_S3_GROUP_KEY = "accounts/groups/%s" % TEST_ACCOUNT_NAME
+TEST_ACCOUNT_S3_USER_KEY = "accounts/users/{account_name}"
+TEST_ACCOUNT_S3_GROUP_KEY = "accounts/groups/{account_name}"
 TEST_ACCOUNT_S3_KEYS = [TEST_ACCOUNT_S3_USER_KEY, TEST_ACCOUNT_S3_GROUP_KEY]
-CREATE_CMD = "disco_accounts.py adduser --name %s --password password" % TEST_ACCOUNT_NAME
-EDIT_CMD = "disco_accounts.py edituser --name %s" % TEST_ACCOUNT_NAME
-DISABLE_CMD = "disco_accounts.py edituser --name %s --active no" % TEST_ACCOUNT_NAME
+CREATE_CMD = "disco_accounts.py adduser --name {account_name} --password password"
+EDIT_CMD = "disco_accounts.py edituser --name {account_name}"
+DISABLE_CMD = "disco_accounts.py edituser --name {account_name} --active no"
 REMOVE_CMDS = ["disco_creds.py delete --key %s" % _key for _key in TEST_ACCOUNT_S3_KEYS]
 
 
 class DiscoAccountsTests(IntegrationTest):
     """
     Tests bin/disco_accounts.py
     """
+    _multiprocess_shared_ = True
```

Comment:
Could `_multiprocess_shared_ = True` be moved to the base class `IntegrationTest`